scripts/installers/docker-setup.sh

```diff
@@ -69,7 +69,7 @@ function pull_image() {
 function setup_mountpoints() {
     echo -e "${RED}Creating mount points${NC}"
     for dev in /dev/sr?; do
-        sudo -u arm mkdir -p "/mnt$dev"
+        sudo mkdir -p "/mnt$dev"
```

Comment:
Great catch! What this command was trying to do is make sure that `/mnt/dev/sr#` is owned by arm`, but it fails when run. Your fix succeeds, but leaves `/mnt/dev/sr#` owned by `root`, see below.